features/environment.py

```diff
@@ -870,15 +879,18 @@ def create_and_set_output_directory(self, feature_name):
         os.makedirs(feature_dir)
         self._output_dir = feature_dir
 
-    def clone(self, from_name, cluster_name, to_name):
+    def clone(self, from_name, cluster_name, to_name, long_running=False):
         f = self._processes[from_name]
+        max_wait_limit = -1 if long_running else 10
```

Comment:
I think 10 is too small value.
Default value in PatroniPoolController.start() is 40, and we shouldn't decrease it.